client/pages/editcard.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+export default class EditCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    const value = event.target.value;
+    if (event.target.name === 'question') {
+      this.setState({ question: value });
+    } else {
+      this.setState({ answer: value });
+    }
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const cardId = this.props.cardId;
+    const req = {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    };
+    fetch(`/api/card/${cardId}`, req)
+      .then(res => {
+        res.json();
```

Comment:
Sweet, removed the arrow function since it was unnecessary

![image](https://user-images.githubusercontent.com/109567499/200958754-3d8615a1-e009-4019-80ec-151e63dcc300.png)